VideoPlusChat/src/main/java/com/example/videochatcomposesample/ui/theme/Theme.kt

```diff
@@ -0,0 +1,53 @@
+package com.example.videochatcomposesample.ui.theme
+
+import androidx.compose.foundation.isSystemInDarkTheme
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.darkColors
+import androidx.compose.material.lightColors
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.graphics.Color
+
+private val DarkColorPalette = darkColors(
+    primary = Purple500,
+    primaryVariant = LightGray,
+    secondary = DarkGray,
+    background = Color.Black,
+    surface = Color.Black,
+    onPrimary = Color.White,
+    onSecondary = Color.White,
+    onBackground = Color.White,
+    onSurface = Color.White,
+
+)
+
+private val LightColorPalette = lightColors(
+    primary = Purple500,
+    primaryVariant = LightGray,
+    secondary = DarkGray,
+    background = Color.Black,
+    surface = Color.Black,
+    onPrimary = Color.White,
+    onSecondary = Color.White,
+    onBackground = Color.White,
+    onSurface = Color.White,
+
+)
+
+@Composable
+fun VideoChatComposeSampleTheme(
+    darkTheme: Boolean = isSystemInDarkTheme(),
+    content: @Composable () -> Unit
+) {
+    val colors = // if (darkTheme) {
```

Comment:
If the code in the comments is not used, please delete it. We always can come back to it through the version system :-)